lib/functions.php

```diff
@@ -722,6 +722,22 @@ function simplesaml_check_force_authentication() {
 		// no need to do anything if already logged in
 		return;
 	}
+
```

Comment:
I think the cidr check should be just before the standard force check on line 758.. just to keep related checks together, but also because the disable_sso check should go first, just to quick return before some more intensive function calls